public/js/scene.js

```diff
@@ -22,6 +22,11 @@ export class Scene {
         this.game.appendChild(this.menu.getMenu());
     }
 
+    aogOpenMainMenu() {
+        this.game.innerHTML = "";
```

Comment:
NIT: Change this to the RemoveChild method iteratively. Setting innerHTML like that has potential to cause issues with the rest of the app.